modules/integrate-with-qcr-scanner.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-vulnerability-scanners.adoc
+:_module-type: PROCEDURE
+[id="integrate-with-qcr-scanner_{context}"]
+= Integrating with Quay Container Registry
+
+You can integrate {product-title} with Quay Container Registry for scanning images.
+
+.Prerequisites
+* You need an *OAuth Token* for authentication with the Quay Container Registry.
```

Comment:
You could change "You need" to "You have" for consistency with the other prerequisites in this assembly.